src/components/Drawer/index.js

```diff
@@ -25,10 +25,11 @@ export default class Drawer extends React.Component {
       PropTypes.node,
       PropTypes.func,
       PropTypes.string
-    ])
+    ]),
+    withGradient: PropTypes.bool
```

Comment:
Can you please rename this prop to `withSpotLight`, so it is consistent with the `Header` component's API? This way we can also apply different gradients in the future.